content/themes/casper

```diff
@@ -1 +1 @@
-Subproject commit de91a4006b49868aa5ffc1f7af3b79abe1595897
+Subproject commit 1c092c7be098bed496eaa2f194a3ae39dd23659d
```

Comment:
I think the way to keep this out is to make sure your local copy of master is up to date, and then run `git rebase master` on the branch this PR is based from.